### PR TITLE
More cookie fortunes

### PR DESCRIPTION
**Pull request by lapatison**
## About the PR

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
Screenshot_1

**Changelog**

:cl:
- add: Over 50 new fortunes can be found in fortune cookie

### CODE REVIEW

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  - Head of your deparment is proud of you!
+  - Ian is proud of you!
+  - Do it for Ian!
+  - Always carry an emergency box.
+  - Red spy is in the base.
+  - In the coming minutes, your cherished desire will come true.
+  - Take a quick look in the mirror!
+  - Don't look in the mirror today!
+  - Friendship will come to you soon.
+  - Grey man is a gas mask will come to you soon.
```
Comment (lapatison) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:37]: typo to fix

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  - Don't look back.
+  - Never trust a person in squeaky shoes.
+  - Never trust a person in red uniform.
+  - Never trust a person in green uniform.
+  - Never trust a person in blue uniform.
+  - Never trust a person in yellow uniform.
+  - Never trust a person in black uniform.
+  - Never trust a person without uniform.
+  - Always trust a person in squeaky shoes.
+  - Clown knows that you know that he knows. Beware.
```
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:16]:
```suggestion
  - The clown knows that you know that he knows. Beware.
```

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  - Beware of silent man in silent maints.
```
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:17]:
```suggestion
  - Beware of the silent man in the silent maintenance tunnels.
```

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  - Be careful in dark maintenance tunnels.
+  - A surprise awaits you in maintenance tunnels.
```
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:19]:
```suggestion
  - A surprise awaits you in the maintenance tunnels.
```

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  - You will be pleasantly surprised in dormitory.
+  - You will be pleasantly surprised in cafeteria.
+  - You will be pleasantly surprised in arrivals.
+  - You will be pleasantly surprised in church.
+  - You will be pleasantly surprised in tool storage.
```
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:24]: 'the dormitory', 'the cafetaria', maybe 'the tool storage'

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+  - There is an impostor among us.
+  - A fun adventure awaits you in maintenance tunnels.
```
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:26]:
```suggestion
  - A fun adventure awaits you in the maintenance tunnels.
```

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+  - CentCom is proud of you!
+  - Head of your deparment is proud of you!
```
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:28]:
```suggestion
  - The head of your deparment is proud of you!
```

File: Resources/Prototypes/Datasets/Names/fortunes.yml (hunk 1 above)
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:37]: 'the' or 'a' grey man

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+  - Always carry an emergency box.
+  - Red spy is in the base.
+  - In the coming minutes, your cherished desire will come true.
+  - Take a quick look in the mirror!
+  - Don't look in the mirror today!
+  - Friendship will come to you soon.
+  - Grey man is a gas mask will come to you soon.
+  - Whoever gets this note will receive a chocolate bar.
+  - Milk is good for your bones.
+  - Shout "Hurray!" and get insulate gloves.
```
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:40]:
```suggestion
  - Shout "Hurray!" and get insulated gloves.
```

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+  - Hug your captain and you will receive a gift.
+  - A dubious friend may be an enemy in red camouflage.
+  - A faithful colleague is a strong defense.
+  - A friend asks only for your time not your crowbar.
```
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:44]:
```suggestion
  - A friend asks only for your time, not your crowbar.
```

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+  - Your life doesn't get better by chance, it gets better by doing your job properly.
+  - Boo!
+  - The real treasure is the friends you made along the shift.
+  - Allan please add details
+  - You are the greatest person who ever lived!
+  - NanoTrasen locked me to this desk and is forcing me to make fortunes for these cookies please help!
```
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:50]:
```suggestion
  - NanoTrasen is forcing me to make fortunes for these cookies, please help!
```

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+  - All your hard work will soon pay off.
+  - Don’t just think, act!
+  - An apple a day keeps the doctor away.
+  - Don't drink and set up singulo.
+  - Stare into the singulo, and the singulo stares back at you.
+  - Every good and perfect gift is from CentCom.
+  - There’s no such thing as Space Station 13.
+  - There is a time for caution, but not for fear.
+  - The smart thing to do is to begin trusting your intuitions.
+  - Time is precious, but insulated gloves are more precious than time
```
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:65]:
```suggestion
  - Time is precious, but insulated gloves are more precious than time.
```

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+  - Fortune is a giver and a taker. Do it.
+  - Fortune and flowers do not last forever. Do it.
+  - Do it. Just do it.
+  - When life gives you lemons instead of oranges, make lemonade.
+  - By opening this cookie you’ve set a terrible chain of events in motion, good job.
+  - Get back to work!
+  - Cookie.print not found
```
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:72]:
```suggestion
  - Cookie.print() not found
```

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+  - THERE IS NO CLONER ON THIS STATION
+  - Putting suit sensors on max is always a good idea.
+  - Syndicate is hiring. Contact us.
+  - The man next to you is up to something. You know what you have to do.
```
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:76]:
```suggestion
  - The person next to you is up to something. You know what you have to do.
```

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+  - Did you know? Monkeys are not what they seem.
+  - Did you know that half of salvage specialists go missing on job?
```
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:78]:
```suggestion
  - Did you know that half of salvage specialists go missing on the job?
```

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 15]
```diff
@@ -0,0 +1,82 @@
...
+  - Also try out sashimi with galaxythystle sauce!
+  - Did you know that 70% of Nanotransen applicants go missing within the first years of their contract? The more you know!!
+  - This shift you are gonna do a very robust move in front of everyone.
```
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:81]:
```suggestion
  - This shift you are going to do a very robust move in front of everyone.
```

File: Resources/Prototypes/Datasets/Names/fortunes.yml (hunk 3 above)
Comment (moonheart08) [reply Resources/Prototypes/Datasets/Names/fortunes.yml:17]: This flows worse, why?

Comment (Visne) [reply Resources/Prototypes/Datasets/Names/fortunes.yml:17]: Because it seems like incorrect grammar to me.

Comment (lapatison) [reply Resources/Prototypes/Datasets/Names/fortunes.yml]: English is not my first language, so I'll rely on Visne suggestions;
Many thanks.

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 16]
```diff
@@ -62,21 +62,21 @@
...
-  - Cookie.print not found
+  - Cookie.print() not found
   - THERE IS NO CLONER ON THIS STATION
   - Putting suit sensors on max is always a good idea.
   - Syndicate is hiring. Contact us.
-  - The man next to you is up to something. You know what you have to do.
+  - The person next to you is up to something. You know what you have to do.
   - Did you know? Monkeys are not what they seem.
-  - Did you know that half of salvage specialists go missing on job?
+  - Did you know that half of salvage specialists go missing on thr job?
```
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:78]:
```suggestion
  - Did you know that half of salvage specialists go missing on the job?
```

File: Resources/Prototypes/Datasets/Names/fortunes.yml [hunk 17]
```diff
@@ -13,19 +13,19 @@
...
-  - A surprise awaits you in maintenance tunnels.
-  - You will be pleasantly surprised in dormitory.
-  - You will be pleasantly surprised in cafeteria.
-  - You will be pleasantly surprised in arrivals.
-  - You will be pleasantly surprised in church.
-  - You will be pleasantly surprised in tool storage.
+  - A surprise awaits you in the maintenance tunnels.
+  - You will be pleasantly surprised in the dormitory.
+  - You will be pleasantly surprised in the cafeteria.
+  - You will be pleasantly surprised in the arrivals.
```
Comment (Visne) [inline Resources/Prototypes/Datasets/Names/fortunes.yml:22]:
```suggestion
  - You will be pleasantly surprised in arrivals.
```
not this one

**Review by Visne — CHANGES_REQUESTED** [review]

**Review by Visne — APPROVED** [review]

**Comment by Visne** [comment]
LGTM after fixing tests

**Pull request merged**